website/docs/deployment.mdx

```diff
@@ -90,41 +90,41 @@ export default function Home() {
 
 There are a few common hosting options:
 
-- [Self hosting](#self-hosting) with an HTTP server like Apache2 or Nginx;
-- Jamstack providers, e.g. [Netlify](#deploying-to-netlify) and [Vercel](#deploying-to-vercel). We will use them as references, but the same reasoning can apply to other providers.
-- [GitHub Pages](#deploying-to-github-pages). (By definition, it is also Jamstack, but we compare it separately.)
+- [Self hosting](#self-hosting) with an HTTP server like Apache2 or Nginx
+- Jamstack providers (e.g. [Netlify](#deploying-to-netlify) and [Vercel](#deploying-to-vercel)). We will use them as references, but the same reasoning can apply to other providers.
+- [GitHub Pages](#deploying-to-github-pages) (By definition, it is also Jamstack, but we compare it separately).
 
 If you are unsure of which one to choose, ask the following questions:
 
 <details>
 
 <summary>
-  How much resource (person-hours, money) am I willing to invest in this?
+  How much time and money am I willing to invest in this?
```

Comment:
"Time and money" is non-exhaustive, so I prefer the wording before, possibly adding "etc." or "e.g.".

Reply:
How about:

How many resources (e.g. money, person-hours, etc.) am I willing to invest in this?

Reply:
Sure. I usually avoid both "e.g." and "etc." so you can choose to omit either one.

Reply:
Good point; I'll take out the e.g.